src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Configuration/PlatformProjectConfigurationDimensionProvider.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.Configuration
     [Export(typeof(IProjectConfigurationDimensionsProvider))]
     [AppliesTo(ProjectCapabilities.ProjectConfigurationsDeclaredDimensions)]
     [Order(DimensionProviderOrder.Platform)]
+    [ConfigurationDimensionDescription(ConfigurationGeneral.PlatformProperty)]
```

Comment:
We don't need a `ConfigurationDimensionDescription` for `TargetFramework`?

Reply:
Already have one, because we were opting into "isVariantDimension", but only for that reason. We added it for Platform/Configuration because CPS uses it to avoid calling us when the solution already has a default for those dimensions - solution doesn't have a default for TargetFramework.